1-python/python_basics/control_structures.ipynb

```diff
@@ -419,14 +417,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "[27 * x for x in range(1, 20) if x <= 10]"
+    "[27 * i for i in range(1, 20) if x <= 10]"
```

Comment:
Ah, I missed the `if`.
Changed it because this is the naming in the `for` loop above. I wanted to use the same names to make it easier to see how they relate to each other.